lib/repositories/message.js

```diff
@@ -394,20 +412,89 @@ function createApi(table, provider, sqlGenerator, dependencies) {
                 });
             });
         });
+    },
+
+    /**
+     * Deletes all messages belonging to the specific mailbox
+     */
+    removeByMailbox: function remove(mailbox) {
```

Comment:
So I noticed I duplicated this construct from the normal remove function... is there any meaning to having 
function function_name(args)
like in this example instead of just
function (args)
like what we do elsewhere?